stix2elevator/cli.py

```diff
@@ -97,7 +97,7 @@ def _get_arg_parser(is_script=True):
     parser.add_argument(
         "--validator-args",
         help="Arguments to pass to stix2-validator. Default: --strict-types\n\n"
-             "Example: stix2_elevator.py <file> --validator-args=\"-v --strict-types -d 212\"",
+             "Example: stix2_elevator.py <file> --validator-args=\"-v 2.0 --strict-types -d 212\"",
```

Comment:
-v is just for verbose, --version is to specify a stix version

```suggestion
             "Example: stix2_elevator.py <file> --validator-args=\"-v --version 2.0 --strict-types -d 212\"",
```

Reply:
I don't remember why I did that, but it is unnecessary